server/container/cassandra-guice/src/main/java/org/apache/james/jmap/MethodsModule.java

```diff
@@ -54,6 +61,16 @@ protected void configure() {
         methods.addBinding().to(new TypeLiteral<GetMessagesMethod<CassandraId>>(){});
         methods.addBinding().to(new TypeLiteral<SetMessagesMethod<CassandraId>>(){});
         bind(SetMessagesUpdateProcessor.class).to(new TypeLiteral<SetMessagesUpdateProcessor<CassandraId>>(){});
```

Comment:
- omitted on purpose, as it required more refactoring work. Has been added separately in following commits, .
- default scope should be fine, as they are injected into consumer classes. No need for singletons I guess.